lib/chef/knife/client_create.rb

```diff
@@ -35,7 +35,7 @@ class ClientCreate < Knife
       option :admin,
              short: "-a",
              long: "--admin",
-             description: "Open Source Chef Server 11 only. Create the client as an admin.",
+             description: "Open Source #{Chef::Dist::SERVER_PRODUCT} 11 only. Create the client as an admin.",
```

Comment:
Looks like I missed removing this feature.

Reply:
I'ma resolve this, I saw you removed this in a different PR